tests/parser/AquiferTests.cpp

```diff
@@ -566,6 +566,104 @@ PORO
     BOOST_CHECK( conf == conf2 );
 }
 
+BOOST_AUTO_TEST_CASE(Test_Aquifer_Config_Active)
+{
```

Comment:
Just a comment, although the current implementation does not forbid it, not sure whether we can have both numerical aquifer and analytical aquifer in the same case. It is clearly stated that we can not have CT aquifer and Fetkovich aquifer together.

Reply:
> although the current implementation does not forbid it, not sure whether we can have both numerical aquifer and analytical aquifer in the same case

Right.  For now I'm just using the implementation's ability to have multiple aquifer types at the same time to reduce the number of decks we have to embed in the unit test files.

> It is clearly stated that we can not have CT aquifer and Fetkovich aquifer together.

Hm, I don't really see that.  There is a definite restriction against using the `AQUFET` and `AQUCT` keywords in the same simulation model, but our Fetkovich model is based on the `AQUFETP` (trailing `P`) and there's no such explicit restriction against combining `AQUFETP` with `AQUCT`.